patterns/footer-centered.php

```diff
@@ -29,7 +29,7 @@
 	<p class="has-text-align-center has-small-font-size">
 		<?php
 		printf(
-			/* Translators: Designed with WordPress. %s: WordPress link. */
+			/* translators: Designed with WordPress. %s: WordPress link. */
```

Comment:
Capitalized "Translators:" apparently works correctly, too.
https://translate.wordpress.org/projects/wp-themes/twentytwentyfive/fr/default/?filters%5Bterm%5D=Designed+with

Reply:
Interesting. Maybe there are other tools where it does not work. I don't know the reason behind this requirement.